tapeagents/mcp.py

```diff
@@ -24,19 +25,33 @@ class NoTool(Exception):
 
 
 class MCPClient:
```

Comment:
Now that I know what this does, I'd rename it to `MCPServersLauncher`.. but that's just an opinion.